src/pubsub/mod_pubsub_db_rdbms.erl

```diff
@@ -266,15 +266,17 @@ set_node(#pubsub_node{nodeid = {Key, Name}, id = undefined, type = Type,
                       owners = Owners, options = Opts, parents = Parents}) ->
     OwnersJid = [jid:to_binary(Owner) || Owner <- Owners],
     {ok, Nidx} = mod_pubsub_db_rdbms_sql:insert_pubsub_node(encode_key(Key), Name, Type,
-                                                     jsx:encode(OwnersJid),
-                                                     jsx:encode(Opts)),
+                                                     jiffy:encode({OwnersJid}),
+                                                     jiffy:encode({Opts})),
```

Comment:
We probably don't need surrounding `{}` for `Opts` either, do we?

Reply:
I think we do. See:
https://github.com/talentdeficit/jsx#to-convert-an-erlang-term-into-a-utf8-binary-containing-a-json-string
```erlang
1> jsx:encode([{<<"library">>,<<"jsx">>},{<<"awesome">>,true}]).
<<"{\"library\": \"jsx\", \"awesome\": true}">>
```
https://github.com/davisp/jiffy#usage
```erlang
2> Doc = {[{foo, [<<"bing">>, 2.3, true]}]}.
{[{foo,[<<"bing">>,2.3,true]}]}
3> jiffy:encode(Doc).
<<"{\"foo\":[\"bing\",2.3,true]}">>
```